src/pages/BlogPage/index.tsx

```diff
@@ -0,0 +1,266 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from 'react-i18next';
+import useStyles from "./style";
+import useQuery from "../../hooks/useQuery";
+import { Grid, Typography, Link, Button, Breadcrumbs } from "@material-ui/core";
+import Footer from "../../components/Footer";
+import ReactMarkdown from "react-markdown";
+import { readingTime } from "../../utils/readingTime";
+import { useViewport } from "../../hooks/viewportWidth";
+import { VIEW_PORT } from "../../constants";
+import BlogsSlider from "../../components/BlogsSlider";
+import Newsletter from "../../components/Newsletter";
+
+const BlogPage: React.FC = () => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const [metadata, setMetadata] = useState<any>();
```

Comment:
Please initialize the state with a value.

Reply:
Done

Reply:
This is not needed now. Removed it